packages/cli/changelog.md

```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next
+
+### Added
+- Contributing guidelines ([#553](https://github.com/zeppelinos/zos/pull/553))
+- Example using truffle migrations ([#416](https://github.com/zeppelinos/zos/pull/416))
+- Integration tests using `truffle-hdwallet-provider` ([#473](https://github.com/zeppelinos/zos/pull/473))
```

Comment:
This is not a new feature, but a fix to integration tests that already used it but were failing